docs/AWS.md

```diff
@@ -0,0 +1,147 @@
+# Running calico-docker on AWS
+Calico runs on the Amazon Web Services (AWS), but there are a few tweaks required to the main Getting Started instructions.  The following instructions show the full power of the Calico routing and security model on AWS (and allow AWS to be used for testing).
+
```

Comment:
The following instructions show how to network containers using Calico routing and the Calico security model on AWS.